filetests/isa/x86/windows_fastcall_x64.clif

```diff
@@ -44,14 +44,16 @@ block0(v0: f64, v1: f64, v2: f64, v3: f64):
 ; check: block0(v0: f64 [%xmm0], v1: f64 [%xmm1], v2: f64 [%xmm2], v3: f64 [%xmm3], v6: i64 [%rbp], v8: f64 [%xmm6], v9: f64 [%xmm7]):
 ; nextln:                         x86_push v6
 ; nextln:                         copy_special %rsp -> %rbp
+; nextln:                         adjust_sp_down_imm 64
```

Comment:
the missing stack adjustment in this test maybe should have been a giveaway that something was awry too. I'd read it as not necessary, assuming `ss0` would be some valid stack slot, but in reality this was testing that a store to `v7` would clobber the start of the CSR region :(

`ss0` is still the correct stack slot with this change because I add the FPR stack slot before the GPR stack slot. I've since updated this test to verify that.